src/Anon.cc

```diff
@@ -1,14 +1,14 @@
-#include <stdlib.h>
-#include <unistd.h>
-#include <assert.h>
-#include <sys/time.h>
-
+#include "Anon.h"
 #include "util.h"
 #include "net_util.h"
-#include "Anon.h"
 #include "Val.h"
 #include "NetVar.h"
 
+#include <stdlib.h>
```

Comment:
The style guide specifies that system headers should come before all other headers (see https://github.com/zeek/zeek-docs/blob/master/devel/contributors/coding-style.rst#including-files). I'm inclined to adjust that to let the class header come before them. Is there another reason for moving the system headers to the end?

Reply:
`Foo.cc` should always include `Foo.h` first to validate that `Foo.h` is self-containing and can be included by others without preconditions.
From there, it seems natural to me that "custom first, system last" is the natural order. We know/assert that system headers must be correct, so we include them in the order they need to be validated. But that's just a subjective opinion, and if it violates your coding style, I'll change it.

Reply:
The style is more along the lines of include-what-you-use, meaning if the source file needs it, it shouldn't be relying on some other header to include it. The easiest way to ensure that is the put them first. If some other header includes it, then include guards will prevent the double-include. Does that make enough sense?

I'll make a note to update the style document to put including the class header first.